app/lib/services/handle_submission_for_store.rb

```diff
@@ -10,8 +10,6 @@ def initialize(store:)
 
     def call
       ActiveRecord::Base.transaction do
-        user.update!(user_update_data)
```

Comment:
This happens in the after_save on qualified_teacher_check now.

Reply:
All users get data set via the GAI login, then any users that would've been updated by this go through qualified_teacher check. Anything that could be set on the user has already been set previously.